src/main/java/io/confluent/kafkarest/ProducerPool.java

```diff
@@ -52,16 +52,15 @@
   private Map<EmbeddedFormat, RestProducer> producers =
       new HashMap<EmbeddedFormat, RestProducer>();
 
-  public ProducerPool(KafkaRestConfig appConfig, ZkUtils zkUtils) {
-    this(appConfig, zkUtils, null);
+  public ProducerPool(KafkaRestConfig appConfig) {
+    this(appConfig, null);
   }
 
   public ProducerPool(
       KafkaRestConfig appConfig,
-      ZkUtils zkUtils,
       Properties producerConfigOverrides
   ) {
-    this(appConfig, getBootstrapBrokers(zkUtils), producerConfigOverrides);
```

Comment:
you can remove the `getBootstrapBrokers` method from this class as it's now in the config class.